scripts/image_generator.sh

```diff
@@ -83,6 +101,7 @@ main()
   python3 "${SCRIPTPATH}"/prepare_bitnami_tags.py
   gen_main_readme
   gen_image_readme
```

Comment:
Can we add both in the same bash function?

gen_image_readme()
For image in image.lst
[ If image exists in image-locked.lst: we use locked jinja template]
[else: use usual one]